File: programmers/Level. 1/문자열 내 p와 y의 개수.py
# 프로그래머스 Lv.1 문자열 내 p와 y의 개수

def solution(s):
    p = 0 # p의 개수
    y = 0 # y의 개수
    
    for i in s :
        if i in ['p', 'P'] :
            p += 1
        elif i in ['y', 'Y'] :
            y += 1
    
    if p == y :
        return True
    else :
        return False

File: programmers/Level. 1/문자열 내 p와 y의 개수.cpp
// 프로그래머스 Lv.1 문자열 내 p와 y의 개수

#include <string>
#include <iostream>
using namespace std;

bool solution(string s)
{
    int p = 0, y = 0; // p와 y의 개수
    
    for (char c : s) {
        if (c == 'p' || c == 'P')
            p += 1;
        else if (c == 'y' || c == 'Y')
            y += 1;
    }
    
    return p == y;
}